tmt/utils.py

```diff
@@ -80,7 +80,7 @@ def relative_to(self, other: Union[str, 'Path']) -> 'Path':  # type: ignore[over
     # ignore[override]: does not match the signature on purpose, our use is
     # slightly less generic that what pathlib supports, to be usable with
     # os.path.relpath.
-    def is_relative_to(self, other: 'Path') -> bool:  # type: ignore[override]
+    def is_relative_to(self, other: 'Path') -> bool:
```

Comment:
If `ignore` is no longer needed, drop also the explanatory comment above (lines 80-82).

Reply:
It's still being used above